[FEAT] Updating some infura doc (#3137)
status-im/nim-beacon-chain: docs/the_nimbus_book/src/infura-guide.md

If you're connecting to a testnet, in the `KEYS` section, click on the dropdown menu to the right of `ENDPOINTS`, and select `GÖRLI`

![](https://i.imgur.com/MKZ6rVc.png)

### 7. Copy one of the endpoints

> You can use either endpoint, but the https one will result in significantly fewer requests used, especially when initially syncing the node.

Reviewer: ```suggestion
You can use either endpoint, but the `https` endpoint will result in significantly fewer requests, especially during initial sync.
```
